ang/civicase/ActivityFeed.html

```diff
@@ -1,12 +1,11 @@
 <div class="panel panel-default civicase__activity-feed">
-  <div class="panel-header" civicase-activity-filters="filters"></div>
+  <div civicase-activity-filters="filters" case-timelines="caseTimelines" display-options="displayOptions"></div>
+
   <div class="panel-body clearfix">
     <div crm-ui-debug="activities"></div>
     <!-- activity details -->
-    <div class="act-feed-view-activity" civicase-activity-view="viewingActivity" refresh-callback="refreshAll" ng-class="{'cc-zero-w': !viewingActivity.id, 'cc-zero-h': !viewingActivity.id}"></div>
+    <!-- <div class="act-feed-view-activity" civicase-activity-view="viewingActivity" refresh-callback="refreshAll" ng-class="{'cc-zero-w': !viewingActivity.id, 'cc-zero-h': !viewingActivity.id}"></div> -->
```

Comment:
This is the Activity Details Panel. I have commented this as the UI for this looks broken. This will be fixed in the next ticket C51-82. But I can uncomment this. let me know.

Reply:
Uncommented.